src/containers/Tenant/Query/QueryEditor/QueryEditor.tsx

```diff
@@ -335,7 +325,7 @@ function QueryEditor(props: QueryEditorProps) {
             <QueryEditorControls
                 handleSendExecuteClick={handleSendExecuteClick}
                 onSettingsButtonClick={handleSettingsClick}
-                isLoading={explainQueryResult.isLoading || executeQueryResult.isLoading}
+                isLoading={Boolean(executeQuery.result?.isLoading)}
```

Comment:
why only `executeQuery`?

Reply:
executeQuery.result can be both execute or explain type